src/main/java/MyRunnable.java

```diff
@@ -0,0 +1,30 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import org.apache.log4j.Logger;
+
+public class MyRunnable implements Runnable {
+    private static final Logger logger = Logger.getLogger(MyRunnable.class);
+    private static final String PATH_FILE = "src/main/resources/test.txt";
```

Comment:
```suggestion
    private static final Logger logger = Logger.getLogger(MyRunnable.class);
```
we don't need this if we have logger